### PR TITLE
Update fields in items getting mixed up

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #74

### CODE REVIEW

File: src/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { createUniqueKeyObject } from './utils';
+
+/**
+ * createUniqueKeyObject
+ */
+describe('createUniqueKeyObject', () => {
+  it('Returns an empty object for an empty input array', () => {
```
Comment (asimonok) [inline src/utils.test.ts:7]: please update test names to `should ...`

**Review by asimonok — CHANGES_REQUESTED** [review]

File: src/utils.test.ts (hunk 1 above)
Comment (vitPinchuk) [reply src/utils.test.ts]: test names updated with 'Should' word

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — CHANGES_REQUESTED** [review]
Some records in Volkov Labs YouTube channel and Grafana RSS feed has empty spaces and displayed incorrectly:

Screenshot 2024-05-26 at 1 24 22 PM

**Comment by vitPinchuk** [comment]
@mikhail-vl  ready for review

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**